src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -535,9 +534,9 @@ public DetailedPackageMetadata PackageMetadata
                     OnPropertyChanged(nameof(PackageMetadata));
                     OnPropertyChanged(nameof(IsPackageDeprecated));
                     OnPropertyChanged(nameof(IsPackageVulnerable));
-                    OnPropertyChanged(nameof(IsPackageVulnerableOrDeprecated));
                     OnPropertyChanged(nameof(PackageVulnerabilityCount));
                     OnPropertyChanged(nameof(PackageVulnerabilities));
+                    OnPropertyChanged(nameof(IsPackageVulnerableOrDeprecated));
```

Comment:
If another code path sets PackageVulnerabilities, that code path would also need to make sure to update IsPackageVulnerable, PackageVulnerabilityCount, PackageVulnerabilities, IsPackageVulnerableOrDeprecated. I think these could all be updated only in the setter for PackageVulnerabilities, instead of here. Thoughts?

Reply:
Also, the issue I mentioned in my previous comment has not been resolved here. For example, take the IsPackageVulnerable property. In this code path, the PackageVulnerabilities will be set which will invoke `OnPropertyChanged(nameof(IsPackageVulnerable));` then after setting PackageVulnerabilities in this method, we once again call `OnPropertyChanged(nameof(IsPackageVulnerable));` therefore calling it twice. We should remove that redundancy.

Reply:
I agree, I'm going to move most of these items to the setter for PackageVulnerabilities. That said I want to keep IsPackageVulnerableOrDeprecated in this location since it needs to be updated if the Deprecation information changes. Since that isn't set anywhere else right now, I want to keep here. If I move it to PackageVulnerabilities then we'll fire it twice.